src/test/resources/acceptance/features/DeleteAJourney.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Delete a journey
+
+  As a driver, I want to delete a journey
+
+  Scenario Outline: Delete a journey
+    Given journey with the following details:
+      |id           | 28356590-332e-43e0-ba7c-50c6a98e41a8  |
```

Comment:
this should be in two lines, not in columns I think

Reply:
it's like the others